test/unit/metrics/test_metricframe_bootstrap_values.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+# This collection of tests is to verify the values computed by
+# the bootstrap functionality of MetricFrame. It is based
+# on the fact that if the metric is accuracy for a binary
+# classification problem, then we can use the binomial
+# distribution and approximate it with a gaussian.
+
+import math
+
+import numpy as np
+import pytest
+
+from sklearn.metrics import accuracy_score
+
+from fairlearn.metrics import MetricFrame
+
+# The setup is slightly unorthodx, to allow for splitting of
```

Comment:
Comments welcome on this pattern, since I've not used it before. I'm trying to avoid having a huge test case with lots of assertions in it, but at the same time, having a bunch of stuff critical to the test cases defined at the top of the file like this isn't ideal either.